cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/RecordWriter.java

```diff
@@ -84,6 +84,18 @@ public class RecordWriter
     private final CqlTable cqlTable;
     private StreamSession<?> streamSession = null;
 
+    /**
+     * Constructor that accepts a BulkWriterConfig and constructs the context on the executor.
+     * This is used when the config is broadcast to executors.
+     *
+     * @param config      immutable configuration broadcast from driver
+     * @param columnNames column names array
+     */
+    public RecordWriter(BulkWriterConfig config, String[] columnNames)
+    {
+        this(BulkWriterContext.from(config), columnNames);
```

Comment:
We only create a record writer from the executor, the other constructor is not necessary.

Reply:
true. Deleted.